lessons/07-html-forms/index.html

```diff
@@ -0,0 +1,63 @@
+<html>
+    <head>
+        <title>
+            Create User
+        </title>
+    </head>
+    <body>
+        <main>
+            <h1>Create User</h1>
+            <hr>
+            <form action="create-user.html" method="GET">
+                <label>Email:</label>
+                <input type="email" name="email" placeholder="Email" required/>
+                <br>
+
+                <label>UserName:</label>
+                <input type="text" name="username" placeholder="username" size="30" maxlength="12"/>
+                <br>
+                
+                <label>Password:</label>
+                <input type="text" name="password" placeholder="password" size="30" maxlength="8" required/>
+                <br>
```

Comment:
Instead of separating these with a `<br />`, a better way to do it would be to group each label and input together using a `<div>`

```html
<div>
  <label>Email:</label>
  <input type="email" name="email" placeholder="Email" required/>
</div>

<div>
  <label>UserName:</label>
  <input type="text" name="username" placeholder="username" size="30" maxlength="12"/>
</div>

...etc
```